components/build-service/base/build-pipeline-config/build-pipeline-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: build-pipeline-config
+  namespace: build-service
+data:
+config.yaml: |
+  pipelines:
```

Comment:
Does it make sense to have default pipeline specified, so UI can show it by default? E.g.
```yaml
  default-pipeline-name: docker-builder
  pipelines:
     ...
```

Reply:
added